src/widgets/modals/ByProgramModal.js

```diff
@@ -1,4 +1,5 @@
 /* eslint-disable react/forbid-prop-types */
+/* eslint-disable import/no-named-as-default */
```

Comment:
Bit nicer to just fix the import - just putting `{ ModalContainer }` on the import

Reply:
Yeah, didn't really want to touch files that aren't in scope (`ModalContainer` component is only default exported). Not a big worry! ffd584e08ae86b2c3562b9aaf4175965361ffc7b.